sdk/workloadorchestration/Azure.ResourceManager.WorkloadOrchestration/tests/Azure.ResourceManager.WorkloadOrchestration.Tests.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\src\Azure.ResourceManager.WorkloadOrchestration.csproj" />
+    <ProjectReference Include="$(AzureCoreTestFramework)" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="Resources\**\*.*">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.Core.TestFramework" />
+    <PackageReference Include="Azure.ResourceManager.Resources" />
+  </ItemGroup>
```

Comment:
```suggestion
```

This is not the correct way to reference these.   You're already including the Azure Core test framework on L8,